modules/node-shared/src/main/resources/application.conf

```diff
@@ -182,3 +182,10 @@ delegated-staking {
 metagraphs-sync {
  max-unapplied-global-change-ordinals: 100
 }
+
+price-oracle {
+  # null means all metagraph ids are allowed
+  allowed-metagraph-ids = null
```

Comment:
I think this may already have come up before -- but just to confirm, we need a way to handle verifying the correct metagraph id functionality works on the testnet ticket. Since we need to wait for the other API endpoints to be merged to do the full deployment, I think this PR is okay to merge. But please account for this in the JIRA ticket, since we need to verify allowed metagraph ids is tested on testnet -- if there's already another way to override this, then that's fine as part of a custom deployment -- but we'll need a separate PR for the mainnet ids and that should be captured somewhere (either in ticket or separate PR)